components/action-mgt/org.wso2.carbon.identity.action.execution/src/main/java/org/wso2/carbon/identity/action/execution/impl/ActionExecutorServiceImpl.java

```diff
@@ -140,7 +147,7 @@ public ActionExecutionStatus execute(ActionType actionType, String[] actionIdLis
         } catch (ActionExecutionRuntimeException e) {
             DIAGNOSTIC_LOGGER.logSkippedActionExecution(actionType);
             LOG.debug("Skip executing actions for action type: " + actionType.name(), e);
-            return new ActionExecutionStatus(ActionExecutionStatus.Status.FAILED, eventContext);
+            return new ErrorStatus(new Error(SERVER_ERROR_CODE, SERVER_ERROR_MESSAGE));
```

Comment:
Refer my comment above

Reply:
This is fixed with 98f55c27eab24621c985d7109a911f761c146612